pandas/io/parsers.py

```diff
@@ -3782,3 +3684,59 @@ def _make_reader(self, f):
             self.skiprows,
             self.infer_nrows,
         )
+
+
+def _check_defaults_read(read_kwds: dict, defaults: dict):
+    """Check default values of input parameters, read a line file."""
```

Comment:
this should *return* a new dict of the paramaters, it should actually do the reading; leave that to the caller.

Reply:
Ok